### DISCUSSION TITLE
Increase oxygen tank time for Mech Suit

### DISCUSSION THREAD

**Discussion by MarkCringan** [FakeFishGames/Barotrauma#10922] [Ideas and suggestions]
Simple: Abyssal suit gets increased oxygen time, why not the Mech suit too?

**Comment by BiffW** [comment]
A person wearing Exosuit uses oxygenite tank as much as wearing a Abyss suit. But for oxygen tank, it's as much as wearing a regular suit.
I think that is an issue.
